docs/build/isc/v1.0.0-rc.6/docs/explanations/invocation.md

```diff
@@ -84,10 +86,16 @@ The funds to cover the gas used will be charged directly from the user's on-chai
 
 ## Allowance
 
-Any funds sent to the chain via on-ledger requests are credited to the sender's account.
+An allowance is a security feature within ERC20 token contracts that controls how much one address can spend on behalf
```

Comment:
Doesn't this make it sound like it is only related to ERC20. There is no connection in this text from ERC20 allowance to "our" allowance 🤔

Reply:
How about?
```suggestion
An allowance is a feature within smart contracts that controls how much one address can spend on behalf
```

Reply:
Yeah that works I think